css/OpenBox.css

```diff
@@ -20,7 +20,7 @@
     background-color: #fefefe;
 	
     margin-left: 25%;
-	margin-top: 25%;
+	margin-top: 14%;
```

Comment:
I don't think this will be a permanent fix, as it'll mess up with another screen size.

try looking into margin:-left: auto; and margin-right: auto.